.github/workflows/release.yml

```diff
@@ -303,10 +303,11 @@ jobs:
       run: |
         sudo dpkg -i dpkg.deb
         /usr/bin/enarx --version
+    - run: mv dpkg.deb "enarx_${{ steps.version.outputs.version }}_${{ matrix.architecture.debarch }}.deb"
```

Comment:
Why? This seems to just be extra step, which obfuscates the implementation. If you wanted to rename this, you should rename the `dpkg` directory instead containing the RPM source